stackit/internal/services/ske/ske_acc_test.go

```diff
@@ -50,6 +50,8 @@ var clusterResource = map[string]string{
 	"extensions_acl_cidrs":                             "192.168.0.0/24",
 	"extensions_argus_enabled":                         "false",
 	"extensions_argus_instance_id":                     "aaaaaaaa-1111-2222-3333-444444444444", // A not-existing Argus ID let the creation time-out.
+	"extensions_dns_enabled":                           "true",
+	"extensions_dns_zones":                             "foo.onstackit.cloud",
```

Comment:
We could also just create a simple DNS zone in the test and use it for the extension

Reply:
As discussed, we will use a dummy value for now.